<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level</class>
 <widget class="QDialog" name="Level">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="easyBtm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </widget>
  <widget class="QToolButton" name="mediumBtm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </widget>
  <widget class="QToolButton" name="hardBtm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
